### ISSUE TITLE
[Thermal Expansion] [Feature request] Ability to change Igenous Extruder's cobble generation mode to consume lava

### ISSUE THREAD

**Issue by utoc** [CoFH/Feedback#719]
I would like to request a config option for TE's Igneous Extruder to consume lava when making cobble.  Ideally it would be nice if it was adjustable but if that cannot be done I would like to increase this to 15mb per cobble and further increase stone to 30mb.  Pretty please?  Something like the following:

```
RecipeManagers {
    Extruder {
        B:ConsumeLavaForCobble=true
        I:LavaUsedPerCobble=15 #In millibuckets, has no use if ConsumeLavaForCobble is false
        I:LavaUsedPerStone=30 #mb
        I:LavaUsedPerObsidian=1000 #mb
    }
}
```

**Comment by KingLemming** [comment]
Pondering on this one. Originally, I had removed it because the machine was going to get some alternate uses and a more complex recipe handler. However, I think I'm backpedaling on that, so this is reasonable.

**Comment by Zsashas** [comment]
I'm curious as to why? Cobble doesn't consume the lava when being created in the world.

**Comment by Dulciphi** [comment]
I'm a month late to this but I've only just found it - after possibly trying to do the same thing the requester's doing. I've disabled the ability for lava and a water source block to create cobble, so no more infinite cobble gen. In my case this is to make Ex Nihilo feasible in a non-skyblock world, without making mining and exploration redundant. 

I've removed other sources of infinite/free cobblegen but I balked at removing the Igneous Extruder, hoping there might be a way to turn lava *source blocks* into cobble/stone.

In summary, I'd love a feature like this.
